src/MAUI/Maui.Samples/Samples/Data/EditFeatureAttachments/EditFeatureAttachments.xaml.cs

```diff
@@ -295,48 +297,121 @@ private Task<Stream> GetImageStreamAsync()
             // Present UIImagePickerController.
             UIWindow window = UIApplication.SharedApplication.KeyWindow;
             var viewController = window.RootViewController;
-            viewController.PresentModalViewController(_imagePicker, true);
+            viewController.PresentViewController(_imagePicker, true, null);
 
             // Return Task object.
-            _taskCompletionSource = new TaskCompletionSource<Stream>();
+            _taskCompletionSource = new TaskCompletionSource<byte[]>();
             return _taskCompletionSource.Task;
         }
 
-        void OnImagePickerFinishedPickingMedia(object sender, UIImagePickerMediaPickedEventArgs args)
+        private void OnImagePickerFinishedPickingMedia(object sender, UIImagePickerMediaPickedEventArgs args)
         {
             UIImage image = args.EditedImage ?? args.OriginalImage;
```

Comment:
Maybe a comment here could help - I am a bit unsure what's happening on this line.